src/wp-admin/css/forms.css

```diff
@@ -1576,14 +1595,24 @@ table.form-table td .updated p {
 	}
 
 	.form-table.permalink-structure td code {
-		margin-left: 32px;
 		display: inline-block;
 	}
 
+	.form-table.permalink-structure .structure-selection {
+		margin-top: 8px;
+	}
+
+	.form-table.permalink-structure .structure-selection .row div {
```

Comment:
One more small change, for consistency:
`.form-table.permalink-structure .structure-selection .row > div`

Reply:
Thanks, should be resolved now.